dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/DataverseUserPage.java

```diff
@@ -239,10 +247,6 @@ public void changePassword(ActionEvent e) {
         editMode = EditMode.CHANGE_PASSWORD;
     }
 
-    public void forgotPassword(ActionEvent e) {
```

Comment:
We can't forget password now :P?

Reply:
Yes - forgetting is illegal :)
Jokes aside:
Currently resetting password is done in separate page (`passwordreset.xhtml`)
So in `loginpage.xhtml` we have a direct link to `<a href="passwordreset.xhtml">`
When we enter valid email - then reset password url will be send to to our account (PasswordResetPage.sendPasswordResetLink()).
Url to reset password will be in form `http://<server>/passwordreset.xhtml?token=<token>`
And update of the password will be handled on passwordreset.xhtml page itself (PasswordResetPage.resetPassword())
This method seems to be some kind of leftover (maybe from how password resetting was handled in older versions of dataverse) - couldn't find any invocation of this method